Documentation/RELEASING.md

```diff
@@ -69,41 +69,68 @@ When doing a Swift Protobuf library release:
       _Note:_ This uses that local copy of `SwiftProtobuf.podspec`, but checks
       against the sources on github.
 
-## Protoc Artifactbundle Releases
+## Updating Protobuf and Abseil Submodules
 
-Protoc artifactbundle releases are independent of Swift Protobuf library releases and follow
-a `protoc-vX.Y` naming convention that matches the upstream protoc version.
+The swift-protobuf repository uses git submodules for protobuf and abseil-cpp dependencies.
+These should be updated when new versions are released upstream.
 
-### Creating a protoc release
+### When to update
 
-1. **Trigger the workflow**
+- When a new protobuf release is available from [protocolbuffers/protobuf](https://github.com/protocolbuffers/protobuf/releases)
+- When the abseil-cpp version referenced by protobuf changes
 
-   Go to the [Actions tab](https://github.com/apple/swift-protobuf/actions/workflows/draft_release_protoc_artifactbundle.yml)
-   and manually run the "Draft release protoc artifactbundle" workflow.
+### Updating the submodules
 
-1. **What the workflow does automatically**
+1. **Check for new protobuf releases**
 
-   The workflow will:
-   - Check the latest protoc version from protocolbuffers/protobuf
-   - Check if we already have a matching `protoc-vX.Y` release
-   - If versions differ or no release exists:
-     - Download protoc binaries for all supported platforms
-     - Create a Swift Package Manager compatible artifact bundle
-     - Create a new draft release tagged `protoc-vX.Y`
-     - Upload the artifactbundle to the draft release
-   - If versions match, exit early (no action needed)
+   Visit the [protobuf releases page](https://github.com/protocolbuffers/protobuf/releases)
+   to check if a new version is available.
 
-1. **Publish the release**
+2. **Update the protobuf submodule**
```

Comment:
fyi - you can level these all as `1.` and when rendered they will be correct, that way edits never have to renumber.